api/api-filtering.mdx

```diff
@@ -17,13 +17,13 @@ To use API Filtering, you pass a subset of CEL expressions to the `filter` query
 This example request fetches a list of all your Cloud and Agent endpoints.
 
 ```http
-GET /endpoints?filter='obj.type == "cloud" || obj.type == "agent"'
+GET /endpoints?filter=obj.type == "cloud" || obj.type == "agent"
```

Comment:
do you think it'd be helpful if we had a full valid curl example at the beginning of the doc like this:

```
curl --location 'https://api.ngrok.com/endpoints' \
  --get \
  --data-urlencode '<filter>' \
  --header 'Ngrok-Version: 2' \
  --header 'Authorization: Bearer <token>'
```

 and then shortened these examples to just 
`filter=obj.type == "cloud" || obj.type == "agent"` ?